### PR TITLE
Remove outdated Sprout tests

### PR DESCRIPTION
**Pull request by sellout**
These tests included operations that are no longer supported. Future changes will break them, so the tests have either been removed or updated as appropriate.

### CODE REVIEW

File: qa/rpc-tests/wallet_listnotes.py [hunk 1]
```diff
@@ -88,19 +56,19 @@ def run_test(self):
         assert_equal(len(unspent_tx), 2)
         # sort low-to-high by amount (order of returned entries is not guaranteed)
         unspent_tx = sorted(unspent_tx, key=lambda k: k['amount'])
-        assert_equal(False,             unspent_tx[0]['change'])
```
Comment (nuttycom) [inline qa/rpc-tests/wallet_listnotes.py:91]: These reorderings seem trivial; did I do those? If so I don't think there was a good reason, and they should probably be reverted.

**Review by nuttycom — APPROVED** [review]
utACK

**Review by nuttycom — CHANGES_REQUESTED** [review]
Looks like a number of tests are failing now that deprecated methods have been disabled?

**Review by nuttycom — APPROVED** [review]
utACK https://github.com/zcash/zcash/pull/6316/commits/86c019fdfb4480415586a6fd04a6514c721843f2

**Review by str4d — APPROVED** [review]
utACK

**Pull request merged**